db/migrate/20220117151612_add_alternate_content_sources.rb

```diff
@@ -0,0 +1,44 @@
+class AddAlternateContentSources < ActiveRecord::Migration[6.0]
+  def up
+    create_table :katello_alternate_content_sources do |t|
+      t.string :name, limit: 255
```

Comment:
null: false here too i assume?

Reply:
Yep yep

Reply:
Actually, I'm going to be removing `null: false` in some places because the future "product" ACS type will not use all of the fields that custom ACSs will use.